cfgov/jinja2/v1/youth_employment_success/budget-form.html

```diff
@@ -37,59 +37,56 @@ <h3>
       </div>
     </div>
 
-    <div class="content-l content-l__large-gutters block block__sub">
-      <div class="content-l_col content-l_col-1 u-mobile-reorder">
-        <div class="content-l u-mobile-reorder__second">
-          <label for="yes-money-spent" class="a-label content-l_col content-l_col-1-3 m-right-justify u-hide-on-mobile"">
-            <b>Now, subtract your monthy expenses:</b>
-            <span class="content-r">
-              <b>(-)</b>
-            </span>
+    
+    <div class="content-l content-l__large-gutters block block__sub block__flush-bottom">        
+      <div class="content-l_col content-l_col-1">
+        <div class="content-l u-mobile-reorder">
+          <label for="yes-money-spent" class="a-label content-l_col content-l_col-1-3 m-right-justify u-mobile-reorder__first">
+            Now, subtract your monthy expenses:
+            <span class="h3 content-r u-hide-on-mobile">(-)</span>
           </label>
-          <div class="content-l_col content-l_col-2-3">
-            <b class="u-show-on-mobile">( - )</b>
-            <input type="text" id="yes-money-spent" name="money-spent" class="a-text-input o-yes-budget-spent" disabled="" aria-labelledby="mobile-spent-label">
+          <div class="content-l_col content-l_col-2-3 u-mobile-reorder__third">
+            <span class="h3 u-show-on-mobile" style="margin-right: 10px;">( - )</span>
+            <input type="text" id="yes-money-spent" name="money-spent" class="a-text-input o-yes-budget-spent" disabled="" aria-labelledby="mobile-spent-label" placeholder="$">
           </div>
-        </div>
-        <div>
-        <div class="u-mobile-reorder__first">
-          <div class="content-l">
-            <div class="content-l_col content-l_col-1-3">
-              <p class="a-label_helper">
-                Don’t include transportation costs, but some other sources might be:
-              </p>
-              <ul class="a-label_helper">
-                <li>Rent and utilities</li>
-                <li>Groceries and eating out</li>
-                <li>Credit card bills</li>
-                <li>Student loans</li>
-                <li>Childcare</li>
-                <li>Cell phone</li>
-              </ul>
+        
+          <div class="u-mobile-reorder__second">
+            
+              <div class="content-l_col content-l_col-1-3">
+                <p class="a-label_helper">
```

Comment:
It isn't because an `a-label` nested within the `content-l_col` was producing undesirable styles on mobile.

Is it better to use the `a-label` class and add a custom class override?

Reply:
Hmm, yes probably, since this isn't proper BEM

Reply:
great, I'll open a new PR with the changes!